src/xrCore/xrMemory.h

```diff
@@ -11,7 +11,9 @@
 Поэтому всё-же стоит переопределять для большинства случаев операторы new и delete.
 А для остального мы будем полагать (и надеяться), что прокси справится без проблем.
 */
+#if defined(WINDOWS) // Не знаю, как это на виндах работает, но на линуксе ломается линковка из-за множественного объявления __TBB_malloc_proxy_helper_object
```

Comment:
Плохо, что ломается =(((

Впрочем, TBB итак подцепляется через перегрузки операторов new и собственные функции.

Reply:
IIRC only english comments are acceptable in OpenXRay development. Please rewrite it to something like this:

```
-#if defined(WINDOWS) // Не знаю, как это на виндах работает, но на линуксе ломается линковка из-за множественного объявления __TBB_malloc_proxy_helper_object
+#if defined(WINDOWS) // I have not idea how it works on Windows, but Linux build fails with error 'multiple declaration of __TBB_malloc_proxy_helper_object'
```

Better yet: remove the comment and extract the change to a commit "Fix multiple declaration of '__TBB_malloc_proxy_helper_object' error on Linux"

Better yet: properly fix the problem for all platforms.

Reply:
> Для Windows прокси линкуется со всем, что можно. Visual Studio это переваривает.
В принципе прокси можно линковать только для основного бинарника. Он будет прекрасно работать.
И я думаю это исправит проблему с gcc.

>For Windows TBB proxy is linked to everything. Visual Studio accepts that.
Well, we can link it to main executable only. It would work excellent.
And I think this would fix problem with GCC.

As suggested by @intorr, we can link TBB proxy only to main executable